client/client.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"io"
+	"log"
+	"net"
+	"os"
+)
+
+func main() {
+	addr, err := net.ResolveTCPAddr("tcp", "localhost:8000")
+	if err != nil {
+		log.Fatal(err)
+	}
+	conn, err := net.DialTCP("tcp", nil, addr)
+	if err != nil {
```

Comment:
duplicate nil check here and line 12

Reply:
nvm just saw the reassignment of err var

Reply:
✅